app/controllers/admin/collection_report_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller for creating collection reports
+  class CollectionReportController < Admin::ApplicationController
+    def new
+      authorize!
+
+      @collections = Collection.order(:title)
```

Comment:
Do you need the collections here? They were needed in the item report only to populate the collection selection field.